base_station/gui/src/cam.html

```diff
@@ -0,0 +1,28 @@
+<!doctype html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8"></meta>
+  <meta name="viewport" content="width=device-width"></meta>
+
+  <title>MRover Cameras</title>
+  <link rel="stylesheet" type="text/css" href="/site.css"></link>
+  <link rel="stylesheet" type="text/css" href="/leaflet.css"></link>
+  <style>
+    body {
+      height: 98vh;
+      overflow: hidden;
+    }
+
+    camera {
+      height: 100%;
+    }
+  </style>
+</head>
+
+<body>
+  <camera></camera>

```

Comment:
`<camera>` is invalid HTML. I used `<main>` in index.html because that is a valid HTML element.